### ISSUE TITLE
Relax the colliding keywords constraint for @type

### ISSUE THREAD

**Issue by gkellogg** [w3c/json-ld-api#4]
I want the following:

``` js
{
  "@context": {
    "type": "@type",
    "profile": "@type"
  },
  "type": "cov:Coverage",
  "profile": "cov:GridCoverage"
}
```

However this is not allowed. The playground says "Invalid JSON-LD syntax; colliding keywords detected".

However, this one works:

``` js
{
  "@context": {
    "type": {"@id": "rdf:type", "@type": "@id" },
    "profile": {"@id": "rdf:type", "@type": "@id" }
  },
  "type": "cov:Coverage",
  "profile": "cov:GridCoverage"
}
```

I understand that this restriction probably makes sense for other keywords, but could it do any harm for `@type`?

Original issue [Relax the colliding keywords constraint for @type #402](https://github.com/json-ld/json-ld.org/issues/402).

**Comment by dlongley** [comment]
I don't know of any reason why this wouldn't work for `@type` but maybe someone else does. I wouldn't object to loosening the restriction for `@type` if people think there is a significant need for this.

**Comment by ajs6f** [comment]
I'm probably just expressing my ignorance of the history here, but why was this restriction instituted? It's not obvious to me (yet) what's being guarded against...

**Comment by gkellogg** [comment]
There was discussion in the 1.0 timeframe [here](https://json-ld.org/minutes/2012-04-24/#topic-6), with related issues json-ld/json-ld.org#91 and json-ld/json-ld.org#102. These were getting a slightly different issues, but worth the read to go over the history of the subject.

**Comment by azaroth42** [comment]
Resolution on WG call of 2018-08-17:  We propose using scoped contexts rather than multiple top level aliases for `@type`. Next step is to determine if that meets the original use case or not.

**Comment by iherman** [comment]
This issue was discussed in [a meeting](https://www.w3.org/2018/json-ld-wg/Meetings/Minutes/2019/2019-02-08-json-ld#section5-5).

- `RESOLVED:  Close api#4 wontfix, use case is covered with scoped contexts, and no other known use cases to justify the complexity {: #resolution10 .resolution}`
<details><summary><i class="differentiate">View the transcript</i></summary>
Relax the colliding keywords constraint for @type<br/>
Rob Sanderson: ref: <a href="https://github.com/w3c/json-ld-api/issues/4">https://github.com/w3c/json-ld-api/issues/4</a><br/>
Gregg Kellogg: there is no identified use case for this<br/>
Rob Sanderson: the complications outweigh any benefit<br/>
Proposed resolution: Close api#4 wontfix, use case is covered with scoped contexts, and no other known use cases to justify the complexity (Rob Sanderson)<br/>
Gregg Kellogg: +1<br/>
Ivan Herman: +1<br/>
Rob Sanderson: +1<br/>
Jeff Mixter: +1<br/>
Adam Soroka: +!<br/>
Harold Solbrig: +1<br/>
Benjamin Young: +1<br/>
Simon Steyskal: +1<br/>
David I. Lehn: +1<br/>
David Newbury: +1<br/>
Resolution #10: Close api#4 wontfix, use case is covered with scoped contexts, and no other known use cases to justify the complexity {: #resolution10 .resolution}<br/>
</details>
